app/jobs/add_cards_from_url_job.rb

```diff
@@ -1,16 +1,12 @@
-require "active_support/notifications"
 class AddCardsFromUrlJob < ActiveJob::Base
   queue_as :adding_cards
 
-  # after_perform do |job|
-  #   ActiveSupport::Notifications.instrument("finish.active_job", job) do
-  #     puts "Job finished"
-  #   end
-  # end
+  after_perform do |job|
```

Comment:
Unused block argument - `job`. You can omit the argument if you don't care about it.